src/components/PastTours/PastTourDetails.tsx

```diff
@@ -1,9 +1,94 @@
 import React from 'react'
+import './PastTours.css'
+import question from '../../assets/question-sign.svg'
+import route from '../../assets/travel.svg'
+import lightbulb from '../../assets/light-bulb (1).svg'
 
```

Comment:
I really like these assets. When we settle on our final ui for the form/past tours we should make sure that they look similar.

Reply:
Cool! I was trying to find icons that are similar to the ones in the actual form in the book

Reply:
Agree! They look great.